packages/generator-widget/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Changed
+
```

Comment:
This should also be **Breaking changes** then, right?

Reply:
Since the generator widget produces a new widget each time it is used, is it really a break change if we update the pwt version produced by the gw? It will never interact with a previews generated widget.

I'm really asking, I'm not 100% sure about this one.

Reply:
Ah, you do have a point there :P. Yeah, then we can keep it like this.